diff -urN mono-2.4.2.3/configure.in mono-2.4.2.3.src/configure.in
--- mono-2.4.2.3-orig/configure.in	2009-07-21 23:23:53.000000000 +0200
+++ mono-2.4.2.3/configure.in	2009-10-17 19:24:17.000000000 +0200
@@ -2262,7 +2262,7 @@
   if test "x$cross_compiling" = "xno"; then
     mono_cfg_dir=`cygpath -w -a $mono_cfg_root`\\etc
   else
-    mono_cfg_dir=`echo $mono_cfg_root | tr '/' '\\\'`\\etc
+    mono_cfg_dir=`echo $mono_cfg_root | tr '/' '\\'`\\etc
   fi
 else
   mono_cfg_dir=$mono_cfg_root/etc
